packages/fxa-content-server/app/scripts/lib/cropper.js

```diff
@@ -44,19 +39,34 @@ function Cropper(options) {
   if (!options.container) {
     throw new Error('A container element is required');
   }
-  this._setupElements(options.container);
 
-  if (options.src) {
-    var img = options;
+  setTimeout(() => {
+    this._setupElements(options.container);
 
-    // we want the extra resolution to be able to zoom in later (up to 200%)
-    var maxLength = this.exportLength * 2;
-    if (img.width > maxLength || img.height > maxLength) {
-      var scale = maxLength / Math.max(img.width, img.height);
-      img = this.resize(img, scale);
+    this.updateMeasurements();
+
+    window.addEventListener(
+      'resize',
```

Comment:
I think `Cropper.prototoype.resize` is used to resize the dimensions of and return an image, not a method that is called when a resize event occurs, no?

Reply:
@jodyheavener Ah yep, you are correct, I should have looked more closely!

I checked out your branch again this morning and there's a few things to note...

Having `window.addEventListener('resize',` here creates a new event listener every time the user uploads an image (so every time `new Cropper` is called). We'll want to destroy it when the user closes the modal / destroys the view. Because of this, we may want to capture the event in `avatar_crop.js`. Normally we use Backbone's `events: {` option within the view, but I'm actually having a hard time finding an example on how to stick a window resize event here.

Could we stick the event listener in `avatar_crop.js`'s `initialize()`, remove it in a `remove()` method in that view (which is ran automatically when the view is removed), and communicate this event to the cropper? Maybe there's a simpler approach?

Also, in desktop view, the image is off-center on upload and allows dragging even further to the left.
<img width="300" alt="image" src="https://user-images.githubusercontent.com/13018240/72091521-f9e24580-32d5-11ea-85c2-25069ff98f3d.png">

Additionally, this is also a little buggy when rotating an image, it appears off-center now (desktop & mobile):
<img width="300" alt="image" src="https://user-images.githubusercontent.com/13018240/72091396-b12a8c80-32d5-11ea-8da2-3b6cfee9d25c.png">

It looks like some functional tests are failing too. Are you able to see these locally with the command I provided above?

You are right that this turned into a doozy of an issue😅